docker/docker_entrypoint.sh

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+# load env variables from a file if ENV_FILE is set
+if [[ -n "${ENV_FILE:-}" ]] && [[ -f "${ENV_FILE}" ]]; then
```

Comment:
What is the expected behaviour if both env file and env variables are set? Should we prefer file one over env? Or fail?

Reply:
Expected behaviour is that env file overrides previously set env variables. For now this is what is needed for: https://github.com/Cardinal-Cryptography/aleph-infra/blob/main/initContainer/entrypoint.sh there's no overlap currently.